app/src/main/res/layout/question_player_content_item.xml

```diff
@@ -0,0 +1,32 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+
+  <data>
+
+    <import type="android.view.View" />
+
+    <variable
+      name="htmlContent"
+      type="CharSequence" />
+  </data>
+
+  <FrameLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="12dp"
+    android:layout_marginTop="28dp"
+    android:layout_marginEnd="12dp"
+    android:layout_marginBottom="28dp">
+
+    <TextView
+      android:id="@+id/question_player_content_text_view"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginTop="16dp"
+      android:layout_marginBottom="16dp"
```

Comment:
This should also be removed otherwise the next item will be at `44dp` margin, 16 from textview + 28 from framelayout

Reply:
ditto